presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -297,6 +297,7 @@ public final class SystemSessionProperties
     public static final String NATIVE_AGGREGATION_SPILL_MEMORY_THRESHOLD = "aggregation_spill_memory_threshold";
     public static final String NATIVE_JOIN_SPILL_MEMORY_THRESHOLD = "join_spill_memory_threshold";
     public static final String NATIVE_ORDER_BY_SPILL_MEMORY_THRESHOLD = "order_by_spill_memory_threshold";
+    public static final String NATIVE_WRITER_SPILL_ENABLED = "native_writer_spill_enabled";
```

Comment:
Can you add other existing properties we chat offline here? writer spill hasn't enabled yet.